config/config.go

```diff
@@ -27,14 +35,24 @@ func ParseAsV2_0_0(data []byte) (types.Config, error) {
 		return types.Config{}, err
 	}
 
-	var keyMap map[interface{}]interface{}
-	if err := yaml.Unmarshal(data, &keyMap); err != nil {
-		return types.Config{}, err
-	}
+	var r report.Report
 
-	if err := assertKeysValid(keyMap, reflect.TypeOf(Config{})); err != nil {
-		return types.Config{}, err
+	nodes := yaml.UnmarshalToNode(data)
+	if nodes == nil {
+		fmt.Println("Failed to unmarshal yaml for reporting line numbers.")
```

Comment:
Both fuze's validation and ignition's validation with return reports. Ignition's will not have line numbers since there is no corresponding json source. Should we merge the two and return that? That would lose which source it came from (although we could iterate over the report and add that to the entry messages). 

Another possibility is to refactor so this doesn't return a ignition config structure and instead just parses the yaml and returns a the fuze config structure and fuze report. It's caller can then use that result to call ConvertAs2_0_0. Then it's up to the caller to determine how the reports should be handled.